packages/arb-provider-ethers/src/lib/provider.ts

```diff
@@ -296,11 +296,11 @@ export class ArbProvider extends ethers.providers.BaseProvider {
                     return '0x100';
                 }
                 const arbInfo = ArbInfoFactory.connect(ARB_INFO_ADDRESS, this);
-                return arbInfo.getCode(params.address, { blockTag: params.blockTag });
```

Comment:
Why did you remove the blockTag on these? It's true we ignore it when we actually make the call, but it seemed like leaving it in place would make it easier to implement the blockTag stuff in the future.

Reply:
ah sorry, I wasn't trying to make a change that would affect anything. since the contract interface doesn't describe a second parameter, typescript doesn't compile since it sees an extra parameter. I can put this back and add a `@ts-ignore`?